runtimevar/drivertest/drivertest.go

```diff
@@ -144,14 +156,19 @@ func testString(t *testing.T, newHarness HarnessMaker) {
 	// A second watch should block forever since the value hasn't changed.
 	// A short wait here doesn't guarantee that this is working, but will catch
 	// most problems.
-	tCtx, cancel := context.WithTimeout(ctx, 10*time.Millisecond)
+	tCtx, cancel := context.WithTimeout(ctx, 100*time.Millisecond)
 	defer cancel()
 	got, err = v.Watch(tCtx)
 	if err == nil {
 		t.Errorf("got %v want error", got)
 	}
-	if tCtx.Err() == nil {
-		t.Errorf("got err %v; want Watch to have blocked until context was Done", err)
+	// tCtx should be cancelled. However, tests using record/replay mode can
```

Comment:
This was a potential flake before, that became more obvious when I added another copy of it down below, so I fixed them both.